src/components/shared/IssueTicketsButton.tsx

```diff
@@ -18,18 +30,37 @@ export default function IssueTickets() {
 
     issueTokensTx(
       { name: fields.name, symbol: fields.symbol },
-      { onDone: () => setModalVisible(false) },
+      {
+        onDone: () => setModalVisible(false),
+        onConfirmed: () => {
+          history.go(0)
+          setLoading(false)
+        },
+      },
     )
   }
 
   return (
     <div>
       <Space>
-        <Button loading={loading} onClick={() => setModalVisible(true)}>
-          <Trans>Issue ERC-20 token</Trans>
+        <Button
+          size="small"
+          icon={<SettingOutlined />}
+          loading={loading}
+          onClick={() => setModalVisible(true)}
+        >
+          <span>
```

Comment:
@johnnyd-eth one thing i discovered is that, if you have a button with an icon or a loading state, the `children` have to be either:
- plain text (no `Trans` tag)
- wrapped in a `span`

... or else the icon won't get paddingRight.